metadata.rb

```diff
@@ -2,17 +2,17 @@
 maintainer 'Hearst Automation Team'
 maintainer_email 'atat@hearst.com'
 license 'All rights reserved'
-description 'Installs/Configures incident_bot'
+description 'Installs/Configures the Hearst Incident Bot'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '0.1.0'
+version '1.0.0'
```

Comment:
I don't think the cookbook should have any bearing on another project status should it? If this portion of the project is functional, I would assume if can be an actual release. Also, as this is not served in the traditional sense and is only pulled at CFN run time, version will really not matter in the grand scheme of things.